src/models.jl

```diff
@@ -45,7 +45,7 @@ the type of `model`.
 function get_observation_eltype end
 
 """
-    ParticleDA.sample_initial_state!(state, model, rng)
+    ParticleDA.sample_initial_state!(state, model, rng, task_index)
```

Comment:
Reminder to self to provide some documentation of what `task_index` argument corresponds to and why its present.